### PR TITLE
feat(android): Add support for Android 11, remove Android 9 support

### PR DESCRIPTION
**Pull request by jeromelaban**
GitHub Issue (If applicable): fixes https://github.com/unoplatform/uno/issues/4137

## PR Type

What kind of change does this PR introduce?
- Feature

## What is the new behavior?

BREAKING CHANGE: The support for Android 9 is being removed, as the Play Store does not accept updates for applications using this version.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: src/SourceGenerators/XamlGenerationTests/XamlGenerationTests.csproj [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="MSBuild.Sdk.Extras" ToolsVersion="15.0">
 	<PropertyGroup>
 		<TargetFrameworks>MonoAndroid10.0;xamarinios10;netstandard2.0</TargetFrameworks>
```
Comment (Xiaoy312) [inline src/SourceGenerators/XamlGenerationTests/XamlGenerationTests.csproj:3]: not bumping to 11 here?

Comment (jeromelaban) [reply src/SourceGenerators/XamlGenerationTests/XamlGenerationTests.csproj:3]: No good reason :) Thanks!

File: src/Uno.UI/Controls/CommandBarRenderer.Android.cs [hunk 2]
```diff
@@ -194,7 +194,9 @@ protected override void Render()
 				Native.Menu.Clear();
 				foreach (var command in Element.PrimaryCommands.Concat(Element.SecondaryCommands).OfType<AppBarButton>())
 				{
+#pragma warning disable 618
```
Comment (davidjohnoliver) [inline src/Uno.UI/Controls/CommandBarRenderer.Android.cs:197]: should this be replaced by something else eventually?

Comment (davidjohnoliver) [reply src/Uno.UI/Controls/CommandBarRenderer.Android.cs:197]: ditto for all the rest of the `Obsolete` warning disables. There are quite a few of them...

Comment (jeromelaban) [reply src/Uno.UI/Controls/CommandBarRenderer.Android.cs:197]: It should, yet until this does not break, it's not a good reason to remove it.

**Review by davidjohnoliver — APPROVED** [review]

**Pull request merged**